bindings/ergo-lib-wasm/src/wallet.rs

```diff
@@ -62,6 +68,42 @@ impl Wallet {
             .map(Transaction::from)
     }
 
+    /// Sign a transaction:
```

Comment:
This doc comment is the same as for `sign_transaction` and does not convey what differences does it have.

Reply:
By merging signers this issue is resolved.